scripts/build-style-vars.js

```diff
@@ -60,4 +60,7 @@ async function main() {
     }
 }
 
-main();
+main().catch(e => {
```

Comment:
make sure errors cause a nonzero exit code

Reply:
This should apply to any of our node scripts. We should either:

- use the `sync` versions of any `fs` commands (which would block until completed or error out) and handle errors procedurally
- preserve any async calls as-is, but make sure the script itself contains the proper `.catch` or try-catch block(s)

We could use this PR to address them all. Would require a small audit.

Reply:
interesting, I assumed thrown errors would cause a non-zero error, but good to know